c2corg_api/scripts/migration/init_feed.py

```diff
@@ -15,6 +15,7 @@ def migrate(self):
             self.session_target.execute(SQL_DOCUMENT_CREATE_CHANGE)
             self.session_target.execute(SQL_AREAS_FOR_CHANGES)
             self.session_target.execute(SQL_ACTIVITIES_FOR_CHANGES)
+            self.session_target.exececute(SQL_LANGUAGES_FOR_CHANGES)
```

Comment:
I disagree with that: it's useless. The migration script will no longer be executed and we will no longer apply changes to the migration script when changing the DB model in the future => please remove those changes.

On the other hand don't you need to add an alembic script for the need language attributes in the feed?